Backend/core/comms.py

```diff
@@ -1,14 +1,17 @@
-import json, select, socket, struct
-import sys
+import json, select, socket, struct, sys, time
+import threading
+import traceback
 
 import config
+import requests
```

Comment:
Will need something other than `requesets` (maybe aiohttp) if using asyncio to spawn coroutines for the requests